packages/cashier/src/pages/payment-agent/payment-agent-signup/country-selector/country-selector.tsx

```diff
@@ -0,0 +1,127 @@
+import React from 'react';
+import { connect } from 'Stores/connect';
+import { ResidenceList, CountriesListResponse } from '@deriv/api-types';
```

Comment:
@hamid-deriv could we use useStore instead of connect?

Reply:
Sure, Sorted.